packages/components/psammead-sitewide-links/src/List/index.jsx

```diff
@@ -40,72 +43,87 @@ const StyledList = styled.ul`
   }
   @media (min-width: ${GEL_GROUP_1_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
     grid-column-gap: ${GEL_SPACING};
-    grid-template-columns: repeat(2, 1fr);
-    grid-template-rows: repeat(
-      ${({ children }) => getRowCount(children, 2)},
-      auto
+    grid-template-columns: repeat(
+      ${props => (props.TrustProjectLink ? 3 : 2)},
+      1fr
     );
+    grid-template-rows: repeat(${({ links }) => getRowCount(links, 2)}, auto);
     column-count: 2;
   }
   @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
     grid-column-gap: ${GEL_SPACING_DBL};
-    grid-template-columns: repeat(3, 1fr);
-    grid-template-rows: repeat(
-      ${({ children }) => getRowCount(children, 3)},
-      auto
+    grid-template-columns: repeat(
+      ${props => (props.TrustProjectLink ? 4 : 3)},
```

Comment:
```suggestion
      ${({ TrustProjectLink })=> (TrustProjectLink ? 4 : 3)},
```

lets be consistent with how we access props